charts/traceability-foss-backend/values.yaml

```diff
@@ -111,58 +115,52 @@ ingress:
   hosts: []
   tls: []
 
-##############################
-# Application Configuration  #
-#############################
-
-springprofile: <springprofile>
-
 healthCheck:
   enabled: true #<healthCheck.enabled>
 
 traceability:
-  bpn: <traceability.bpn>
-  url: <traceability.url>
+  bpn: "CHANGEME" #<traceability.bpn>
+  url: "" #<traceability.url>
 
 datasource:
-  url: <datasource.url>
-  username: <datasource.username>
-  password: <datasource.password>
+  url: jdbc:postgresql://tracex-backend-postgresql:5432/trace
+  username: trace
+  password: "CHANGEME" #<datasource.password>
 
 oauth2:
-  clientId: <oauth2.clientId>
-  clientSecret: <oauth2.clientSecret>
-  clientTokenUri: <oauth2.clientTokenUri>
-  jwkSetUri: <oauth2.jwkSetUri>
-  resourceClient: <oauth2.resourceClient>
+  clientId: "CHANGEME" #<oauth2.clientId>
+  clientSecret: "CHANGEME" #<oauth2.clientSecret>
+  clientTokenUri: "https://changeme.com" #<oauth2.clientTokenUri>
+  jwkSetUri: "https://changeme.com" #<oauth2.jwkSetUri>
+  resourceClient: "CHANGEME" #<oauth2.resourceClient>
 
 edc:
-  apiKey: <edc.apiKey>
-  providerUrl: <edc.providerUrl>
+  apiKey: "" #<edc.apiKey>
+  providerUrl: "" #<edc.providerUrl>
 
 postgresql:
-  enabled: false #<postgresql.enabled>
+  enabled: true
+
+  nameOverride: "tracex-backend-postgresql"
+  fullnameOverride: "tracex-backend-postgresql"
+
   auth:
-    postgresPassword: <postgresql.auth.postgresPassword>
-    password: <postgresql.auth.password>
-    database: <postgresql.auth.database>
-    username: <postgresql.auth.username>
-  architecture: standalone
-  audit:
-    pgAuditLog: write, ddl
-    logLinePrefix: "%m %u %d "
+    postgresPassword: "CHANGEME" #<postgresql.auth.postgresPassword>
+    password: "CHANGEME" #postgresql.auth.password>
+    database: "trace"
```

Comment:
why is the database name fix? Why MUST it be trace and what happens if i change the name?